packages/ods/src/components/tag/src/constants/tag-size.ts

```diff
@@ -1,10 +1,9 @@
 enum ODS_TAG_SIZE {
   lg = 'lg',
   md = 'md',
-  sm = 'sm',
```

Comment:
Not in the issue, Is it deliberate ?

Reply:
This is four different issue (one per commit), and yes the size removale is one of them